### PR TITLE
SOC-1483 | Displaying message on empty forums

### PR DESCRIPTION
**Pull request by bkoval**
https://wikia-inc.atlassian.net/browse/SOC-1483

@kvas-damian @rafalkalinski @Bloodmoon24

This will go through design review in the meantime as I didn't add a "plus" icon yet. I'll find out if we need it. Original pull request #1829

### CODE REVIEW

File: front/locales/en/discussion.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
     "no-replies": "No replies yet.",
     "no-replies-be-the-first": "No replies yet. Be the first!",
     "modal-dialog-confirm": "OK",
-    "modal-dialog-log-in": "Sign in"
+    "modal-dialog-log-in": "Sign in",
+    "empty-forum-text": "No posts yet. Get the discussion started, create the first post now!"
```
Comment (kvas-damian) [inline front/locales/en/discussion.json]: I thought we're going to split this message into 2 lines.

Comment (rafalkalinski) [inline front/locales/en/discussion.json]: Please don't forget about translations

File: front/scripts/main/components/discussion-empty-forum-message.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export default Ember.Component.extend({
+	discussionEditor: Ember.inject.service(),
+	classNames: ['empty-forum-message'],
```
Comment (kvas-damian) [inline front/scripts/main/components/discussion-empty-forum-message.js]: Alphabetical order ;)

File: front/styles/main/component/discussion-empty-forum-message.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.discussion {
+	.empty-forum-message {
+		svg {
```
Comment (kvas-damian) [inline front/styles/main/component/discussion-empty-forum-message.scss]: svg block should be moved below attributes.
BTW, please use class name selector

File: front/locales/en/discussion.json (hunk 1 above)
Comment (bkoval) [inline front/locales/en/discussion.json]: OK ;)

**Comment by kvas-damian** [comment]
LGTM

**Comment by rafalkalinski** [comment]
LGTM

**Pull request merged**